### DISCUSSION TITLE
Role Pings? (Some servers really need it)

### DISCUSSION THREAD

**Discussion by randomy1** [stoatchat/discussions#743] [Feature Suggestions]
No, im not saying @everyone or @here pings, im saying ROLE pings, where u ping a certain role & it mentions everyone in that role.

alot of servers need this for announcements. and it would be helpful

**Comment by lemte** [comment]
This really makes server dynamics easier, avoids role creation and the need to mention where using **@everyone** is most practical

**Reply by uhidontkno** [reply]
i literally tried making a role called "everyone" hoping @everyone would work :skull:

**Comment by bmvoss123** [comment]
> No, im not saying `@everyone` or `@here` pings

I'd personally argue `@everyone`/`@here` can be useful in smaller servers. But while we're talking about pings, when selecting from the autocomplete list on trying to ping someone, is there a way that instead of getting the user's hexadecimal unique identifier, can we retain the display name in order to know who we're pinging _after_ the selection is made?
